src/UI/Button.elm

```diff
@@ -266,11 +268,23 @@ fromLabel label =
     Button.fromIcon (Icon.map "Go to maps")
 
 -}
-fromIcon : Icon -> ButtonBody
+fromIcon : Icon -> ButtonBody msg
 fromIcon icon =
     BodyIcon icon
 
 
+fromNested : String -> (String -> Icon) -> ButtonBody msg
+fromNested label icon =
+    BodyNested label icon
+
+
+fromAtomicNest : String -> (String -> Icon) -> msg -> ButtonBody msg
```

Comment:
I didn't like this naming, but why we would need this? I think you added this because you wanted to transform the header in button right? But I don't think it is a button, is a specific clickable view, but is not a button